src/main/java/org/folio/holdingsiq/service/impl/HoldingsRequestHelper.java

```diff
@@ -114,6 +114,24 @@ public HoldingsRequestHelper(Configuration config, Vertx vertx) {
     return future;
   }
 
+  protected CompletableFuture<Void> postHoldingsRequest(String query){
```

Comment:
Is this a general purpose method?
If not don't you think it should be next to its usage